case_studies/shallow_water/checkpoints/galewsky_checkpoints.py

```diff
@@ -0,0 +1,203 @@
+
```

Comment:
can we have a description of what this is useful for at the top?

Reply:
e78df9bdc5dbb8508d2b23621d3751bb12aa5c27